nucleus/payara-modules/hazelcast-bootstrap/src/main/java/fish/payara/nucleus/hazelcast/admin/GetHazelcastConfiguration.java

```diff
@@ -109,6 +109,7 @@ public void execute(AdminCommandContext context) {
         ColumnFormatter columnFormatter = new ColumnFormatter(headers);        
         columnFormatter.addRow(new Object[]{"Configuration File",runtimeConfiguration.getHazelcastConfigurationFile(),"Domain"});
         columnFormatter.addRow(new Object[]{"Interfaces",runtimeConfiguration.getInterface(),"Domain"});
+        columnFormatter.addRow(new Object[]{"Port Auto Increment", runtimeConfiguration.getPortAutoIncrement(), "Domain"});
```

Comment:
"Auto Increment Port" reads a bit better.

Reply:
Its called "Port Auto Increment" in hazelcast so I would think avoiding confusion caused by using a very similar name is more important than something that reads better? WDYT?